src/utils/shallowEqual.js

```diff
@@ -1,3 +1,13 @@
+/**
+ * Given two objects, performs equality by iterating through keys
+ * on an object and returning `false` when any key has values which
+ * are not strictly equal between `objA` and `objB`.
+ * Returns `true` when the values of all keys are strictly equal.
+ *
+ * @param  {Object} objA
+ * @param  {Object} objB
+ * @return {Boolean}
+ */
```

Comment:
Documentation is not the same as the implementation.
Implementation just checks `objA === objB`